ibr/png-parser/create_ibr.py

```diff
@@ -43,7 +47,7 @@
     new_ibr.visualization.append(vis)
```

Comment:
we should probably create wrapper classes for these instead of modifying raw IBR protos